docs/GettingStarted/configuration/ProductionBestPractice.md

```diff
@@ -1,68 +1,58 @@
 # Customizing Pelorus
 
-See [Configuring the Pelorus Stack](../../configuration/PelorusCore/) for a full readout of all possible configuration items. The following sections describe the  most common supported customizations that can be made to the Pelorus configuration object YAML file.
+See [Configuring the Pelorus Stack](./PelorusCore.md) for a full readout of all possible configuration items. The following sections describes the most important configuration options of Pelorus, focusing on the production environment.
 
-## Configure Prometheus Retention
+## Preface
```

Comment:
I would remove the title and everything before it. I think the information is already in the following paragraph

Reply:
changed.